### PR TITLE
commit

### PR DESCRIPTION
**Pull request by anastasia-radchencko**
https://anastasia-radchencko.github.io/react_people-table-advanced/

### CODE REVIEW

File: src/components/PersonLink.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    slug, sex, name, born, died, mother, father, motherName, fatherName,
+  } = person;
+
+  const { slug: slugValue } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': slugValue === slug,
```
Comment (TarasHoliuk) [inline src/components/PersonLink.tsx:24]: it doesn't work
image

File: src/components/PeopleFilters.tsx [hunk 2]
```diff
@@ -27,55 +68,32 @@ export const PeopleFilters = () => {
...
+                {num}
+              </Link>
+            ))}
           </div>
 
           <div className="level-right ml-4">
-            <a
+            <Link
               data-cy="centuryALL"
               className="button is-success is-outlined"
```
Comment (TarasHoliuk) [inline src/components/PeopleFilters.tsx:92]: should be filled if no filters selected

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
You should add routes in the App.tsx to display the correct page:
image
image

File: src/components/PeoplePage.tsx [hunk 3]
```diff
@@ -15,17 +106,27 @@ export const PeoplePage = () => {
...
-                There are no people on the server
-              </p>
+              {!isLoading && people.length === 0 && (
+                <p data-cy="noPeopleMessage">
+                  There are no people on the server
+                </p>
+              )}
 
-              <p>There are no people matching the current search criteria</p>
+              {false && (
```
Comment (etojeDenys) [inline src/components/PeoplePage.tsx:123]: it will be always hidden

**Review by etojeDenys — CHANGES_REQUESTED** [review]
let's try to fix your routes
image



or feel free to ask in chat for some help

**Pull request closed without merging**